apps/api/src/app/shared/shared.module.ts

```diff
@@ -102,7 +102,6 @@ const distributedLockService = {
 };
 
 const PROVIDERS = [
-  inMemoryProviderService,
```

Comment:
@scopsy @p-fernandez 
I'm wondering why can't we just use the DI mechanism to inject the `InMemoryProviderService` to the `CacheService` and `DistributedLockService`?... we have here above the provider object and we do call `useFactory` two times which creates two instances of `InMemoryProviderService`...